### DISCUSSION TITLE
how many runs you execute to calculate the average time ?

### DISCUSSION THREAD

**Discussion by samir-** [geeksblabla/blanat#122] [Q&A]
.

**Comment by aboullaite** [comment]
Everything is open in here https://github.com/geeksblabla/blarun

**Reply by samir-** [reply]
times_to_run is default to 5 but it's not clear if it's this exact number that is being run from the human judge.

**Reply by aboullaite** [reply]
We simply take the result calculated from the script

**Reply by samir-** [reply]
blarun does not shot what args goes into the script. whether times_to_run is set or not.

**Reply by anassajaanan** [reply]
@aboullaite It seems there is no function to test C code in the tester :
<img width="534" alt="image" src="https://github.com/geeksblabla/blanat/assets/114141528/4ea7f1a3-6706-4570-80ea-c356e7b8074e">

but in run_cpp it is using g++ instead of gcc, so how it will test C code:

<img width="829" alt="image" src="https://github.com/geeksblabla/blanat/assets/114141528/2ab90a40-11aa-42dd-a296-8b7814b73c0d">

**Reply by aboullaite** [reply]
Can you open a PR to fix that @anassajaanan ?
